doc/modules/preprocessing.rst

```diff
@@ -474,11 +474,11 @@ then the shape of the transformed input is
     Imputer(add_indicator_features=True, axis=0, copy=True, missing_values='NaN',
         strategy='mean', verbose=0)
     >>> print(imp_with_in.transform(X))                           # doctest: +ELLIPSIS
-    [[ 4.          2.          1.          0.        ]
-     [ 6.          3.66666667  0.          1.        ]
+    [[ 4.          2.          0.          0.        ]
```

Comment:
Under shuffled cross-validation, you can't always be sure what's in your training and what's in your test. That's why allowing them to specify to Imputer may be important. However, to take back what I said, the indicator features won't be any use if they're not seen in training, so it's probably valid not to add a parameter to specify which features.